src/client/components/Terms.tsx

```diff
@@ -62,8 +62,27 @@ export const GuardianTerms = () => (
   </>
 );
 
+export const JobsStandaloneTerms = () => (
+  <div css={terms}>
+    <Text>
+      By proceeding, you agree to our{' '}
```

Comment:
maybe `{' '}` is the react way to insert a space, but you can also use `&nbsp;` html entity ...